checkov/runner_filter.py

```diff
@@ -82,6 +83,7 @@ def __init__(
         self.all_external = all_external
         self.var_files = var_files
         self.skip_cve_package = skip_cve_package
+        self.filtered_policy_ids: List[str] = filtered_policy_ids
```

Comment:
```suggestion
        self.filtered_policy_ids: List[str] = filtered_policy_ids or []
```
that's how you could do easily the `None` check here